packages/components/accordion/stories/accordion.stories.ts

```diff
@@ -32,6 +35,70 @@ export default {
         },
       },
     },
+    headerPaddingX: {
+      control: 'select',
+      options: ['smaller', 'small', 'normal', 'large', 'larger'],
+      description: 'Set horizontal paddings of accordion header',
+      table: {
+        defaultValue: {
+          summary: 'large',
+          detail: 'large (24px)',
+        },
+        type: {
+          summary: paddingVariantsSummary,
+          detail:
+            'smaller(4px) | small(8px) | normal(16px)| large(24px) | larger(32px)',
+        },
+      },
+    },
+    headerPaddingY: {
+      control: 'select',
+      options: ['smaller', 'small', 'normal', 'large', 'larger'],
+      description: 'Set vertical paddings of accordion header',
+      table: {
+        defaultValue: {
+          summary: 'normal',
+          detail: 'normal(16px)',
+        },
+        type: {
+          summary: paddingVariantsSummary,
+          detail:
+            'smaller(4px) | small(8px) | normal(16px)| large(24px) | larger(32px)',
+        },
+      },
+    },
+    contentPaddingX: {
+      control: 'select',
+      options: ['smaller', 'small', 'normal', 'large', 'larger'],
+      description: 'Set horizontal paddings of accordion content',
+      table: {
+        defaultValue: {
+          summary: 'large',
+          detail: 'large (24px)',
+        },
+        type: {
+          summary: paddingVariantsSummary,
+          detail:
+            'smaller(4px) | small(8px) | normal(16px)| large(24px) | larger(32px)',
+        },
+      },
+    },
+    contentPaddingY: {
+      control: 'select',
+      options: ['smaller', 'small', 'normal', 'large', 'larger'],
+      description: 'Set vertical paddings of accordion content',
+      table: {
+        defaultValue: {
+          summary: 'normal',
+          detail: 'normal(16px)',
+        },
+        type: {
+          summary: paddingVariantsSummary,
+          detail:
+            'smaller(4px) | small(8px) | normal(16px)| large(24px) | larger(32px)',
+        },
+      },
+    },
```

Comment:
I think we've to use tokens set up in global PUIK project. Currently we're using Tailwind's token like p-5 or ml-5. @FrancoisQtr your point of view ?